src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
+    const { queryParameters: { indexPatternId } } = state;
+    filterManager.add(field, values, operation, indexPatternId);
```

Comment:
I don't see a problem with actions in context view influencing field popularity.